e2etest/e2etest_suite_test.go

```diff
@@ -50,6 +50,15 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+// use ephemeral port (tcp/8080) for pod, instead of well-known port (tcp/80)
+var servicePodPort uint
+var skipDockerCmd bool
+
+func init() {
```

Comment:
Is this a Gingko convention? I don't see where it is called but I did test and it does all seem to be working correctly.

Reply:
It is golang's convention. see here https://golang.org/doc/effective_go#init

This is used not only here, also kubernetes https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/base.go#L37-L39

Reply:
Ok, gotcha. This seems to be an established pattern. Why can't handleFlags() be used? I presume because we need to call `flag.UintVar` and `flag.BoolVar` early?